geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseIndexAdapter.scala

```diff
@@ -255,9 +255,8 @@ class HBaseIndexAdapter(ds: HBaseDataStore) extends IndexAdapter[HBaseDataStore]
         }
         (cqlFilter ++ indexFilter).sortBy(_._1).map(_._2)
       }
-      lazy val coprocessorOptions: Map[String, String] =
-        Map[String, String](GeoMesaCoprocessor.YieldOpt -> ds.config.coprocessors.yieldPartialResults.toString) ++
-            strategy.index.ds.config.queries.timeout.map(GeoMesaCoprocessor.timeout)
```

Comment:
Doesn't the timeout still need to go to the RS?  Maybe it goes somewhere else...

Reply:
it does, but it only gets added in the CoprocessorBatchScan now, since the timeout is not calculated as an absolute timestamp until CoprocessorQueryPlan.scan is invoked.

Reply:
+1